packages/babel-core/src/transformation/plugin.js

```diff
@@ -1,15 +1,12 @@
 import OptionManager from "./file/options/option-manager";
 import * as messages from "babel-messages";
-import Store from "../store";
 import traverse from "babel-traverse";
 import clone from "lodash/clone";
 
 const GLOBAL_VISITOR_PROPS = ["enter", "exit"];
 
-export default class Plugin extends Store {
```

Comment:
👍  I guess we never had to use .get/.set?

Actually what about https://github.com/babel/babel/blob/7.0/packages/babel-plugin-transform-runtime/src/index.js#L32